src/effects.c

```diff
@@ -1999,20 +1999,76 @@ s32 func_800910E4(Player *player) {
     case ITEM_TRIPLE_MUSHROOM:
     case ITEM_SUPER_MUSHROOM:
         if ((player->effects & 8) != 0) {
-            return 1;
-        }
-        phi_v0 = 0x5F934EC4;
-        goto func_800910E4_label;
+            return TRUE;
+        }
+        phi_v0 = (
+            LIGHTNING_EFFECT|
```

Comment:
Consider making a define for this


`#define MY_EXTENDED_EFFECT LIGHTNING_EFFECT | BLAH | BLAH2`